api_tests/providers/registrations/views/test_registration_provider_schemas.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    RegistrationProviderFactory,
+    AuthUserFactory
+)
+
+from osf.models import RegistrationSchema
+
+from osf.migrations import update_provider_auth_groups
+
+
+@pytest.mark.django_db
+class TestRegistrationProviderSchemas:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def schema(self):
+        return RegistrationSchema.objects.get(name='Prereg Challenge', schema_version=2)
+
+    @pytest.fixture()
+    def out_dated_schema(self):
+        reg_schema = RegistrationSchema(name='Prereg Challenge', schema_version=1)
+        reg_schema.save()
+        return reg_schema
+
+    @pytest.fixture()
+    def invisible_schema(self):
+        reg_schema = RegistrationSchema(name='Test Schema (Invisible)', schema_version=1, visible=False)
+        reg_schema.save()
+        return reg_schema
+
+    @pytest.fixture()
+    def inactive_schema(self):
+        reg_schema = RegistrationSchema(name='Test Schema (Inactive)', schema_version=1, active=False)
+        reg_schema.save()
+        return reg_schema
+
+    @pytest.fixture()
+    def provider(self, schema, out_dated_schema, invisible_schema, inactive_schema):
+        provider = RegistrationProviderFactory()
+        update_provider_auth_groups()
+        provider.schemas.add(*[schema, out_dated_schema, invisible_schema, inactive_schema])
+        provider.save()
+        return provider
+
```

Comment:
Sorry, one more thing: could you add a test (or just the fixture and a check in the existing test) where a second provider also has just prereg challenge version 2 and make sure that works properly? Amanda is keen on verifying that scenario works, so we might as well have it tested.